app/playground/page.tsx

```diff
@@ -1,14 +1,20 @@
 import { AskLLM } from "components/AskLLM"
 import { CopyText } from "components/CopyText"
+import { Quiz } from "components/Quiz"
 
 export default async function Page() {
   return (
     <>
       <AskLLM
-        predefinedPrompt='Generate a comma-separated list of 10 ice cream flavours:'
-        predefinedOutput='Chocolate, Vanilla, Strawberry, Mint Chip, Rocky Road, Cookie Dough, Butter Pecan, Neapolitan, Coffee, Coconut'
+        predefinedPrompt="Generate a comma-separated list of 10 ice cream flavours:"
+        predefinedOutput="Chocolate, Vanilla, Strawberry, Mint Chip, Rocky Road, Cookie Dough, Butter Pecan, Neapolitan, Coffee, Coconut"
+      />
+      <CopyText content="Testing content to copy" />
+
+      <Quiz
```

Comment:
**nitpicking:** Seems like old variant of props here.

Reply:
@devodii it is still here, fails the build.

Reply:
Yeah, cleaned it up a bit :)